design-documents/asynchronous-import/base-extension.md

```diff
@@ -23,16 +26,15 @@ Path is relative from Magento Root folder
 
 ```
 {
```

Comment:
do we think that this can be used in multi-select or multi-files upload?

Reply:
Not really. Again it was build based on practical experience. One file transfer already can be quite resource consuming. Multiple its already risky.
If customer want to have multiple files upload he has BULK API available for this case

Reply:
ok, I was just thinking from UI perspective, sometimes we have multi-select option enabled for small files. But in this case it can be risky and may overcomplicate.

Reply:
Yep, I would not stick to multiple files in MVP in any case. Especially when we have BULK API on-board. But this can be discussed in a future steps if we will get request from users